### ISSUE TITLE
Package debug libraries

### ISSUE THREAD

**Issue by vegardkv** [conda-forge/boost-cpp-feedstock#16]
I am not too familiar with conda-forge, but I just recently discovered this package and conda-installed it, and noticed that the debug libraries are not included. This is an issue at least on Windows, where it is not recommended - and may not even work - to combine the Boost release libraries with a debug build.

When building Boost, this is controlled by the "variant" option AFAIK, which can be set to either debug or release (or both).

After some searching I realize that this issue may also be relevant for the boost-cpp feedstock.

**Comment by jakirkham** [comment]
This came up before in PR ( https://github.com/conda-forge/boost-feedstock/pull/35 ). The conclusion was they should either be in a different package or a different variant of this package. The latter would be preferable from a maintenance perspective. Also whatever we use should work for other Windows debug libraries.

**Comment by vegardkv** [comment]
Right, I had not noticed the pull request. I see that there are indeed valid arguments for not including the debug build as well.

Thank you for the PR reference!

**Comment by jakirkham** [comment]
FWIW this is still a great idea that we should really revive (especially as the tooling has caught up a bit). Raised issue ( https://github.com/conda-forge/conda-forge.github.io/issues/544 ) to discuss how we do split packages and generate things like debug builds as products.
